### ISSUE TITLE
Ability to change icon size

### ISSUE THREAD

**Issue by Pranavraut033** [tjackenpacken/taskbar-groups#113]
I have created 2 groups in my taskbar, extremely helpful but the icons seem a little out of layout.

PS: Custom scaling is set to 125 in my windows

![image](https://user-images.githubusercontent.com/20313943/112934482-7feae600-913f-11eb-91b9-2138a5e6e376.png)
![image](https://user-images.githubusercontent.com/20313943/112934516-95f8a680-913f-11eb-8530-fca884fa10e0.png)

**Comment by PikeNote** [comment]
Hello!

I will try to work on this as a solution but as an alternative, I do want to try something.
I tweaked the settings of the picture box for the application when you have them there to make it possibly automatically fit.

Here is a small beta build based on this master release
[Debug.zip](https://github.com/tjackenpacken/taskbar-groups/files/6226346/Debug.zip)
See if this works. There are some extra files there that are usually hidden to prevent users from accidentally deleting/moving them.

**Comment by Fosternator** [comment]
@Pranavraut033 

You might want to look into: 

Fixed icon and UI on HiDPi-enabled devices. #255 

**Please comment the pull request to give it more traction.** 😄

Also, here's a beta build of the HDPi version:
[TaskbarGroups_HiDpi.zip](https://github.com/tjackenpacken/taskbar-groups/files/10128256/TaskbarGroups_HiDpi.zip)

Have a great day ! 👋 (let me know if it helped)

**Comment by PikeNote** [comment]
@Fosternator 
Hi! If you can, send a pull request on https://github.com/PikeNote/taskbar-groups-pike-beta/
I no longer have contact with the repo owner and cannot merge any pull requests.
Most likely will start on some cleanup work to get everything up and going again.

**Comment by Fosternator** [comment]
@PikeNote 
Hiya! 
Sounds good, I saw your build after posting here (which is awesome too btw), I looked through the different commits and I wasn't sure if you had addressed this issue or not. I'll send a pull request later tonight !

**Comment by PikeNote** [comment]
@Fosternator I don't believe I ever got around to fixing it. I am trying to patch up stuff at the moment. You can check with the new build and if it's not fixed there, then it isn't fixed at all. I commented accidentally on your pull to request a pull onto my repo again because I forgot it was in this thread. I was confused for a moment since I forgot I asked inside an issue.

**Comment by drawbars37** [comment]
@PikeNote I'm coming late to the party, but I also have in interest in this problem/feature.  Like the OP, I am using Win11 at 125% scaling.  The icons are a little small for my liking, both taskbar and Taskbar-Groups (TG).  I can fix the regular taskbar with StartAllBack, but TG insists on using the Windows size -- either 'regular' or 'small icons.'  No 'large icons' in native Windows.

Unlike the OP, I am a retired programmer.  I have almost zero experience with C#, but 30+ years of C programming, as well as JS, HTML/CSS, etc.  (Even assembly code!)  I would be happy to help with this issue (and maybe others).  I've looked at the source code, and while I can understand the basic approach, I'm having trouble finding the details.  (Might be because I started programming in 1974, long before OOP existed!)

Feel free to contact me offline here, or at bw@wahler.us.
